models/reporting/olids/risk_stratification/fct_person_efi2_snapshot_input.sql

```
@@ -0,0 +1,20 @@
+date, = current_date()) and age_at_end (age at that date) -- which advance on every build, so
+snapshotting the full model with check_cols: all would version every person on every run. This
+view exposes only the stable frailty measures (the eFI2 score and its category band), so the
+snapshot versions only on a genuine change (a new score / band). Kept separate from Rockwood to
+mirror how cltcs_cohort_data reads frailty.
+
+Downstream: snapshots/fct_person_efi2_snapshot.yml
+*/
+
+select
+    person_id
+    , efi_score
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)